README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status][build_status_badge]][build_status_link]
 [![PyPI version][pypi_badge]][pypi_link]
 
-Python-dotenv reads key-value pairs from a `.env` file and can set them as environment
+Python-dotenv reads key-value pairs from `.env` files and can set them as environment
```

Comment:
I'm hesitant to change this yet as the core/python API, doesn't support reading from multiple files.

Reply:
Good point. I agree it makes sense to only document this in the CLI section.

Resolved in 1d33b6ad8e09f8bd1ffef73def57b581eafae4b9